lambdatimes/src/index.js

```diff
@@ -3,4 +3,4 @@ import ReactDOM from 'react-dom';
 import './CSS/index.css';
```

Comment:
so in you index file is where you would wrap your app with authenticate, maybe something like:
import Authenticate from "..// authenticate",
const wrappedApp - Authenticate(app)

and in the reactDom.render wrap your app with the "wrappedApp" to complete the HOC